CHANGELOG.md

```diff
@@ -4,6 +4,9 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 This CHANGELOG follows the format located [here](https://github.com/sensu-plugins/community/blob/master/HOW_WE_CHANGELOG.md)
 
 ## [Unreleased]
+### Deprecated
```

Comment:
this is a `### Breaking Changes` rather than a deprecation (which implies still functional and slated for a future removal, see https://github.com/sensu-plugins/community/blob/master/HOW_WE_CHANGELOG.md for more information.

Reply:
Thanks; duly noted. Also, updated.